content/en/docs/tasks/manage-daemon/rollback-daemon-set.md

```diff
@@ -4,29 +4,24 @@ reviewers:
 title: Perform a Rollback on a DaemonSet
 content_type: task
 weight: 20
+min-kubernetes-server-version: 1.7
 ---
 
 <!-- overview -->
 
-This page shows how to perform a rollback on a DaemonSet. 
-
-
+This page shows how to perform a rollback on a {{< glossary_tooltip term_id="daemonset" >}}.
 
 
 ## {{% heading "prerequisites" %}}
 
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
```

Comment:
nit: The text rendered by the include is formatted differently.

Reply:
noticed that too. may need some eyes on it.